include/nbl/video/IGPUPrimaryCommandBuffer.h

```diff
@@ -0,0 +1,50 @@
+#ifndef __IRR_I_GPU_PRIMARY_COMMAND_BUFFER_H_INCLUDED__
+#define __IRR_I_GPU_PRIMARY_COMMAND_BUFFER_H_INCLUDED__
```

Comment:
whats going on why is there a new heaer file?

CC: @Crisspl

Reply:
this file shouldnt exist, we have just one class for primary and secondary cmdbufs

Reply:
okay this is my bad, `IGPUPrimaryCommandBuffer` was used as a base class but was nowhere to be found so I thought you probably forgot to commit it, so I copied it from `master`

Reply:
nuke it